engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -407,12 +424,28 @@ private static void verifyCompleteness(String title, Type type, FBO fbo) {
                 break;
 
             default:
-                logger.error("FBO '" + title + "' generated an unexpected reply from glCheckFramebufferStatusEXT: " + checkFB);
+                logger.error("FBO '" + urn + "' generated an unexpected reply from glCheckFramebufferStatusEXT: " + checkFB);
                 fbo.setStatus(Status.UNEXPECTED);
                 break;
         }
     }
 
+    /**
+     * Returns the content of the color buffer of the FBO "sceneFinal", from GPU memory as a ByteBuffer.
+     * If the FBO "sceneFinal" is unavailable, returns null.
+     *
+     * @return a ByteBuffer or null
+     */
+    public ByteBuffer getSceneFinalRawData() {
```

Comment:
Let's rename this to getColorBufferRawData() and let's update the javadoc as it seems outdated.